ProjectLighthouse/Pages/SlotPage.cshtml

```diff
@@ -31,7 +31,7 @@
     <div class="eight wide column">
         <div class="ui blue segment">
             <h2>Description</h2>
-            <p>@(string.IsNullOrEmpty(Model.Slot.Description) ? "This level has no description." : Model.Slot.Description)</p>
+            <p>@Html.Raw(string.IsNullOrEmpty(Model.Slot.Description) ? "This level has no description." : Model.Slot.Description)</p>
```

Comment:
The usage of `Html.Raw` on user input here makes me a little uncomfortable... 

Is this really the best way to show this?

Reply:
User input should always get sanitized before reaching that but I'll change it to `HttpUtility.HtmlDecode` to be safe